share/completions/terraform.fish

```diff
@@ -0,0 +1,74 @@
+function __fish_terraform_needs_command
+    set cmd (commandline -opc)
```

Comment:
In general you should use local variables to avoid modifying a global var by the same name; i.e., `set -l cmd...`.